package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gasbuddy/react-navigation-codegen",
-  "version": "5.1.4",
```

Comment:
Since we're removing functionality with the top level import, I like this top level major version change